src/_includes/layouts/issue.njk

```diff
@@ -111,7 +111,7 @@
         <div>
             <div class="spacing-y-1">
                 <h2>Description</h2>
-                {{ content | safe }}
+                {{ content | replace(r/\s([A-Z]+\-[0-9]+)/, ' <a href="/browse/$1/">$1</a>') | safe }}

```

Comment:
This should probably be applied to comments as well. See Engage-124. Do you have an example issue of where this linking happens in the description?

Reply:
Here's a couple: https://deploy-preview-16--fluidproject-issues.netlify.app/browse/FLUID-6742/, https://deploy-preview-16--fluidproject-issues.netlify.app/browse/FLUID-6741/

Reply:
It seems that this is only replacing the first instance. In FLUID-6742 there are multiple instances of issues being mentioned, which aren't being converted to links.